package/wpe/wpetvplatform-bcm/Config.in

```diff
@@ -0,0 +1,5 @@
+menuconfig BR2_PACKAGE_TVPLATFORM_BCM
+	bool "wpetvplatform-bcm"
+    depends on (BR2_PACKAGE_BCM_REFSW || BR2_PACKAGE_HAS_NEXUS)
+	help
+	  Base library for TVPlatform Implementation for Broadcom.
```

Comment:
@pwielders did you agree on the wpetvplatform naming? You usually seem to have an opinion about it so cross-checking :)

Reply:
@pwielders , Could you please provide your opinion on this ?